packages/fx-core/templates/plugins/resource/cicd/azdo/README.md

```diff
@@ -0,0 +1,47 @@
+# How to use pre-cooked pipelines on Azure DevOps
+
+## Prerequisites
+- Teams app projects are version controlled by Azure DevOps.
+- An M365 account. If you do not have M365 account, apply one from [M365 developer program](https://developer.microsoft.com/en-us/microsoft-365/dev-program)
+- An Azure service princial with necessary permissions.
+
```

Comment:
better to include the prerequsites of AZDO pipeline.  e.x. make sure you are adminisitrator of your AZDO project to ensure you have the authority to create pipeline.
![image](https://user-images.githubusercontent.com/25972542/157641637-81a855dd-3551-4dce-8bda-f94b9e9e605d.png)